openwisp_monitoring/monitoring/tests/test_notifications.py

```diff
@@ -80,6 +80,29 @@ def test_general_check_threshold_deferred_not_crossed(self):
         self.assertTrue(m.is_healthy)
         self.assertEqual(Notification.objects.count(), 0)
 
+    def test_resources_metric_threshold_deferred_not_crossed(self):
+        # Verify that newly created metrics within threshold limit don't send any notifications
+        self._create_admin()
+        self.create_test_adata()
+        self.assertEqual(Notification.objects.count(), 0)
```

Comment:
can you please add a test for the `threshold_crossed` signal to ensure it's not emitted in this case?